news/10479.feature.rst

```diff
@@ -0,0 +1 @@
+New resolver: When backtracking prefer the dependencies which caused the most recent backtracking. In some cases this will significantly reduce the amount of backtracking required.
```

Comment:
```suggestion
New resolver: When backtracking prefer the dependencies which are involved in the most recent conflict. In some cases this will significantly reduce the amount of backtracking required.
```

Reply:
I have no opinion on this, but FYI I used "New resolver" as that is what was all the resolver news as recently as 21.2.2: https://github.com/pypa/pip/blob/main/NEWS.rst#bug-fixes-2

And my other PR got merged with "New resolver": https://github.com/pypa/pip/pull/10482/files

Happy to change though.